setup/web_server_configuration.rst

```diff
@@ -336,6 +336,13 @@ The **minimum configuration** to get your application running under Nginx is:
     or ``config.php`` scripts (i.e. ``http://example.com/app_dev.php`` and ``http://example.com/config.php``).
     If you *can* access these, be sure to remove the ``DEV`` section from the above configuration.
 
+.. note::
+
+    By default, Symfony applications include several ``.htaccess`` files to
+    configure redirections and to prevent unauthorized access to some sensitive
+    directories. Those files are only useful when using Apache, so you can
+    safely remove them when using Nginx.
```

Comment:
I would also add that they would better be replaced by a virtual host config for performance reasons.